bin/runtime/src/lib.rs

```diff
@@ -754,6 +757,53 @@ impl pallet_identity::Config for Runtime {
     type WeightInfo = pallet_identity::weights::SubstrateWeight<Self>;
 }
 
+parameter_types! {
+    /// We compute the deposit required for an account based on the size of the `AssetAccount`
+    /// struct.
+    pub const AssetAccountDeposit: Balance =
```

Comment:
This is probably quite expensive because of using `LEGACY_DEPOSIT_PER_BYTE`, perhaps using something like `CONTRACT_DEPOSIT_PER_BYTE` would be better, but it's actually tricky to get it right... Here is some methodology for computing these fees https://www.notion.so/cardinalcryptography/Contracts-storage-fee-30f8019d129144a99698136d65f390cc#6aa42d8761bb448da9b6dbfaef19bbcc